Docker_files/ocsci_container/scripts/run-ocs-ci.sh

```diff
@@ -4,5 +4,15 @@ set -e
 mkdir -p /opt/cluster/logs_ocsci_container
 
 # Execute run-ci
-cd /opt/ocs-ci
-run-ci --cluster-path /opt/cluster "$@" > /opt/cluster/logs_ocsci_container/output.txt
+if [[ "$@" == *"--debug--"* ]]; then
+  cd /opt/ocs-ci-debug
+  $PIP_VERSION install --upgrade pip setuptools
+  $PIP_VERSION install setuptools==65.5.0
+  $PIP_VERSION install -r requirements.txt
+  STR_TMP="$@"
+  CMD=${STR_TMP:10:5000000}
+  exec $CMD | tee /opt/cluster/logs_ocsci_container/output_$RANDOM.txt
```

Comment:
Wouldn't be better to use current timestamp instead of random number, like this?
```suggestion
  exec $CMD | tee /opt/cluster/logs_ocsci_container/output_$(date '+%s').txt
```
It will then be easier to find the latest log.

Reply:
done